langchain4j-mcp/src/main/java/dev/langchain4j/mcp/client/transport/http/McpResponseLoggingInterceptor.java

```diff
@@ -0,0 +1,52 @@
+package dev.langchain4j.mcp.client.transport.http;
+
+import java.io.IOException;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import okhttp3.Headers;
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class McpResponseLoggingInterceptor implements Interceptor {
+
+    private static final Logger log = LoggerFactory.getLogger(McpResponseLoggingInterceptor.class);
+
+    @Override
+    public Response intercept(Chain chain) throws IOException {
+        Request request = chain.request();
+        Response response = chain.proceed(request);
+        if (response.code() != 200) {
```

Comment:
This only applies to the initial GET request that starts the SSE channel, and that should return 200, as far as I know, which means that the SSE channel was successfully created. Otherwise, it's an error that should be logged.

Reply:
Ok I'm now realizing that maybe this `McpResponseLoggingInterceptor` isn't necessary at all then, because SSE events are logged anywhere, and if the SSE channel creation fails, the execution doesn't even get here, it ends up in `SseEventListener.onFailure` already